webrecorder/browsertrix
# feat: Allow users to specify language preference

Resolves https://github.com/webrecorder/browsertrix/issues/2119

## Changes

- Shows language selector on log in page
- Adds tabs to account settings
- Adds language preference selector on account settings
- Adds issue template for new language
- Updates dev docs for frontend localization

## Manual testing

1. Go to log in page (or log out.) Verify language selector is shown in app bar
2. Change language to Spanish
3. Log in. Verify archived items navigation label is shown as translated "Archivado"
4. Reload browser window. Verify language change persist
5. Go to account dropdown -> "Account Settings". Verify tabs are shown
6. Change language to English
7. Go to home page. Verify "Archived Items" is shown in English
8. Reload browser window. Verify language change persist

## Screenshots

| Page | Image/video |
| ---- | ----------- |
| Log in | <img width="756" alt="Screenshot 2024-11-05 at 6 46 27 PM" src="https://github.com/user-attachments/assets/d346c88f-3a85-4b4b-988f-6a755360d255"> |
| Archived Items | <img width="1337" alt="Screenshot 2024-11-05 at 6 47 13 PM" src="https://github.com/user-attachments/assets/92ea612a-f136-46d4-9b63-80c9242b4ecf"> |


## Follow-ups

Per Discord convo, we may want a dedicated localization section in the docs in the future.
